content/swiftui/concepts/notifications/notifications.md

```diff
@@ -0,0 +1,190 @@
+---
+Title: 'Notifications'
+Description: 'Learn how to use notifications in SwiftUI for event handling and communication.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'Notifications'
+Table of Contents:
+- Introduction
+- Alerts
+- Action Sheets
+- Toasts (Message Overlays)
+- Custom Notifications
+- Best Practices
+---
+# SwiftUI Notifications
+
+## Introduction 
+
+In **[SwiftUI](https://developer.apple.com/documentation/swiftui/)**, **`notifications`** refer to visual and interactive user interface components that display messages, alerts, or updates to the user. 
+
+They are used to provide information, prompt user interaction, or display important messages within your app's user interface. 
+
+SwiftUI provides several types of notifications that we can use to enhance the user experience. We can use modifiers and view modifiers to create and disply notifications as part of the interface. 
+
+In Swift (using NotificationCenter) notifications are primarly a mechanism for communication between different app components.  They are also visual elements used to present information or gather inputs from the user. 
+
+The two concepts are related and they both involve some form of communication or interaction, but they serve different purposes and are implemented differently in the respective frameworks.
```

Comment:
```suggestion
In SwiftUI, **`notifications`** refer to visual and interactive user interface components that display messages, alerts, or updates to the user. They are used to provide information, prompt user interaction, or display important messages within the app's user interface. 

SwiftUI provides several types of notifications that can be used to enhance the user experience. Modifiers and view modifiers can be used to create and display notifications as part of the interface. In Swift (using NotificationCenter) notifications are primarily a mechanism for communication between different app components.  They are also visual elements used to present information or gather inputs from the user. 

The two concepts are related and they both involve some form of communication or interaction, but they serve different purposes and are implemented differently in the respective frameworks.
```
We don't like linking to external documentation. We also don't use first- and second-person pronouns in Docs.